src/app/containers/ConsentBanner/useConsentBanners/index.js

```diff
@@ -0,0 +1,188 @@
+import { useReducer, useEffect } from 'react';
```

Comment:
This file used to be called `CanonicalLogic/index.js`. Check the diffs here:

https://www.diffchecker.com/cmUgkrVu

https://www.diffchecker.com/PYVug4S2